SoftLayer/CLI/vlan/create.py

```diff
@@ -0,0 +1,40 @@
+"""Order/create a dedicated server."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+from SoftLayer.managers import ordering
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(epilog="See 'slcli server create-options' for valid options.")
+@click.option('--hostname', '-H', required=True, prompt=True, help="Host portion of the FQDN")
+@click.option('--datacenter', '-d', required=True, prompt=True, help="Datacenter shortname")
```

Comment:
As UI has the Pod option too, it would be nice adding that here, then in which case, just one will be required (datacenter or pod I mean).